src/main/java/com/palantir/common/streams/MoreStreams.java

```diff
@@ -30,6 +38,48 @@
     private static final boolean NOT_PARALLEL = false;
 
     /**
+     * Given a {@link Stream<CompletableFuture<U>>}, this function will return a {@link Stream<U>}
```

Comment:
Given a stream of futures, this function will return a blocking stream of results in completion order, looking at most {@code maxParallelism} futures ahead in the stream.

Reply:
done